docker/Dockerfile.gpu

```diff
@@ -1,11 +1,12 @@
-FROM nvidia/cuda:12.2.0-runtime-ubi8 as base
+FROM nvidia/cuda:11.7.1-runtime-ubi8 as base
```

Comment:
Per their docs, pytorch and 11.7 is the current stable combo. And I tested that this combo works. I did not test the 12.2 version, so it might or might not work. But :shrug: that can be a later PR to re-upgrade to 12.2. I'm only confident that 11.7 works.

Reply:
makes sense - these things tend to lag leading edge a bit